provider/cmd/run.go

```diff
@@ -79,15 +86,101 @@ func RunCmd() *cobra.Command {
 		return nil
 	}
 
+	cmd.Flags().String(FlagBidPricingStrategy, "scale", "Pricing strategy to use")
+	if err := viper.BindPFlag(FlagBidPricingStrategy, cmd.Flags().Lookup(FlagBidPricingStrategy)); err != nil {
+		return nil
+	}
+
+	cmd.Flags().Uint64(FlagBidPriceCpuScale, 0, "cpu pricing scale in uakt")
+	if err := viper.BindPFlag(FlagBidPriceCpuScale, cmd.Flags().Lookup(FlagBidPriceCpuScale)); err != nil {
+		return nil
+	}
+	cmd.Flags().Uint64(FlagBidPriceMemoryScale, 0, "memory pricing scale in uakt")
+	if err := viper.BindPFlag(FlagBidPriceMemoryScale, cmd.Flags().Lookup(FlagBidPriceMemoryScale)); err != nil {
+		return nil
+	}
+	cmd.Flags().Uint64(FlagBidPriceStorageScale, 0, "storage pricing scale in uakt")
+	if err := viper.BindPFlag(FlagBidPriceStorageScale, cmd.Flags().Lookup(FlagBidPriceStorageScale)); err != nil {
+		return nil
+	}
+	cmd.Flags().String(FlagBidPriceScriptPath, "", "path to script to run for computing bid price")
+	if err := viper.BindPFlag(FlagBidPriceScriptPath, cmd.Flags().Lookup(FlagBidPriceScriptPath)); err != nil {
+		return nil
+	}
+	cmd.Flags().Uint(FlagBidPriceScriptProcessLimit, 32, "limit to the number of scripts run concurrently for bid pricing")
+	if err := viper.BindPFlag(FlagBidPriceScriptProcessLimit, cmd.Flags().Lookup(FlagBidPriceScriptProcessLimit)); err != nil {
+		return nil
+	}
+
 	return cmd
 }
 
+const BidPricingStrategyScale = "scale"
+const BidPricingStrategyRandomRange = "randomRange"
+const BidPricingStrategyShellScript = "shellScript"
+
+var AllowedBidPricingStrategies = [...]string{
+	BidPricingStrategyScale,
+	BidPricingStrategyRandomRange,
+	BidPricingStrategyShellScript,
+}
```

Comment:
is this used?

Reply:
I'm making a code change that will show this if a correct value isn't given. I'll change it to non-exported since it should never needed anywhere but here.